src/amo/components/DownloadFirefoxButton/index.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import classNames from 'classnames';
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import type { ApiStateType, UserAgentInfoType } from 'core/reducers/api';
+import Button from 'ui/components/Button';
+
+
+type PropTypes = {
+  className?: 'string',
+  i18n: Object,
+  userAgentInfo: UserAgentInfoType,
+}
+
+export const DownloadFirefoxButtonBase = ({
+  className,
+  i18n,
+  userAgentInfo,
+// eslint-disable-next-line indent
```

Comment:
You can merge in master and possibly remove this now

Reply:
I tried and it still failed 😢

I will try once more but yeah eslint is being difficult 😒

Reply:
Oh hey it works now! 😄